@jkotas https://github.com/dotnet/runtime/commit/7745b5ec3db34cd358b26710c0ec32db8b0b23f7 might have broken source-build. Can you please take a look?

> /vmr/src/runtime/src/coreclr/tools/Common/TypeSystem/Common/Utilities/CustomAttributeTypeNameParser.cs(73,85): error CS0246: The type or namespace name 'TypeName' could not be found (are you missing a using directive or an assembly reference?) [/vmr/src/runtime/src/coreclr/tools/aot/ILCompiler.TypeSystem/ILCompiler.TypeSystem.csproj]
    /vmr/src/runtime/src/libraries/Common/src/System/Reflection/TypeNameParser.Helpers.cs(107,31): error CS0246: The type or namespace name 'TypeName' could not be found (are you missing a using directive or an assembly reference?) [/vmr/src/runtime/src/coreclr/tools/aot/ILCompiler.TypeSystem/ILCompiler.TypeSystem.csproj]
    /vmr/src/runtime/src/libraries/Common/src/System/Reflection/TypeNameParser.Helpers.cs(192,50): error CS0246: The type or namespace name 'TypeName' could not be found (are you missing a using directive or an assembly reference?) [/vmr/src/runtime/src/coreclr/tools/aot/ILCompiler.TypeSystem/ILCompiler.TypeSystem.csproj]
    /vmr/src/runtime/src/coreclr/tools/Common/TypeSystem/Common/Utilities/CustomAttributeTypeNameParser.cs(32,33): error CS0246: The type or namespace name 'TypeNameParseOptions' could not be found (are you missing a using directive or an assembly reference?) [/vmr/src/runtime/src/coreclr/tools/aot/ILCompiler.TypeSystem/ILCompiler.TypeSystem.csproj]

https://dev.azure.com/dnceng-public/public/_build/results?buildId=660798&view=logs&j=609589e2-4f74-5576-cdb7-914bcaea778b&t=5ba03121-3cea-598c-07fa-53815ace41ff

--- a/eng/pipelines/templates/steps/vmr-prepare.yml
+++ b/eng/pipelines/templates/steps/vmr-prepare.yml
@@ -7,12 +7,12 @@ parameters:
 
 steps:
   - checkout: vmr
-    displayName: Clone dotnet/dotnet
+    displayName: Clone dotnet/dotnet (TEST)
     path: vmr
     clean: true
 
   - script: |
       git checkout --track origin/${{ parameters.vmrBranch }}
       echo "##vso[task.setvariable variable=vmrBranch]${{ parameters.vmrBranch }}"
-    displayName: Check out ${{ parameters.vmrBranch }}
+    displayName: Check out ${{ parameters.vmrBranch }} TEST
     workingDirectory: $(Agent.BuildDirectory)/vmr

--- a/eng/pipelines/vmr-build-pr.yml
+++ b/eng/pipelines/vmr-build-pr.yml
@@ -52,7 +52,7 @@ resources:
     type: github
     name: dotnet/dotnet
     endpoint: dotnet
-    # TODO: Change back to main when sdk becomes the VMR's origin
+    # TODO: Change back to ${{ variables.VmrBranch }}
     ref: main-sdk
 
 stages:
@@ -60,7 +60,8 @@ stages:
 - ${{ if not(parameters.disableBuild) }}:
   - template: templates/stages/vmr-build.yml
     parameters:
-      vmrBranch: ${{ variables.VmrBranch }}
+      # TODO: Change back to ${{ variables.VmrBranch }}
+      vmrBranch: main-sdk
       isBuiltFromVmr: false
       isSourceOnlyBuild: ${{ variables.isSourceOnlyBuild }}
       ${{ if contains(variables['Build.DefinitionName'], '-full') }}:
@@ -79,5 +80,6 @@ stages:
     jobs:
     - template: templates/jobs/vmr-synchronization.yml
       parameters:
-        vmrBranch: ${{ variables.VmrBranch }}
+        # TODO: Change back to ${{ variables.VmrBranch }}
+        vmrBranch: main-sdk
         noPush: true

--- a/src/Installer/redist-installer/targets/GenerateLayout.targets
+++ b/src/Installer/redist-installer/targets/GenerateLayout.targets
@@ -1,4 +1,4 @@
-<Project>
+<Project TreatAsLocalProperty="PublicBaseURL">
 
   <PropertyGroup>
     <RedistLayoutPath>$(BaseOutputPath)$(Configuration)\dotnet\</RedistLayoutPath>

